docs/platforms/javascript/common/user-feedback/configuration/index.mdx

```diff
@@ -36,12 +36,39 @@ const feedback = feedbackIntegration({
   autoInject: false,
 });
 
+// Create and render the button
 const widget = feedback.createWidget();
-widget.appendToDom();
 
 // Later, when it's time to clean up:
 widget.removeFromDom();
 ```
+```typescript {tabTitle: NextJS}
```

Comment:
should this be React instead of NextJS? Or should we add a more generic React version?

Reply:
I called it Next because of the "client only" comment.

I could do a react version as well (of this and the other example) where we skip the `useEffect()` and everything else is basically the same. Or just rename this... server react is getting more popular esp with v19.

I didn't it written down anywhere that `getFeedback` isn't stubbed/exported in node, so this was a bit surprising to me. I might put up a PR to fix it in the SDK